experiment/riseupvpn/riseupvpn.go

```diff
@@ -285,10 +286,23 @@ func NewExperimentMeasurer(config Config) model.ExperimentMeasurer {
 // Note that this structure is part of the ABI contract with probe-cli
 // therefore we should be careful when changing it.
 type SummaryKeys struct {
-	IsAnomaly bool `json:"-"`
+	APIBlocked      bool `json:"riseupvpn_api_blocked"`
+	ValidCACert     bool `json:"riseupvpn_ca_cert_valid"`
+	FailingGateways int  `json:"riseupvpn_failing_gateways"`
```

Comment:
```suggestion
	APIBlocked      bool `json:"api_blocked"`
	ValidCACert     bool `json:"valid_ca_cert"`
	FailingGateways int  `json:"failing_gateways"`
```

Let's remove the `riseupvpn` prefix also from here.